src/ui/component/copyToClipboard/copyToClipboard.scss

```diff
@@ -1,7 +1,21 @@
+@keyframes show {
```

Comment:
I'm not sure you need an extra `keyframe` here, you could simply use the `transition` rule no?